js/store.js

```diff
@@ -3,6 +3,10 @@ const store = {
     this.addListeners();
   },
 
+  getEmptyObject() {
```

Comment:
added for testing purpose. once I have sinon-chrome setup working, this function will be removed.